css/style.css

```diff
@@ -27,11 +27,10 @@ body {
   color: #000;
   font-size: 12px;
   line-height: 1.4;
-  font-family: Helvetica, Arial, sans-serif;
+  font-family: font-family: 'Open Sans', sans-serif;
```

Comment:
Careful here. You have `font-family` written twice.  This is actually invalid and is preventing `Open Sans` from being applied.